pkg/kapp/app/recorded_app.go

```diff
@@ -94,15 +99,59 @@ func (a *RecordedApp) CreateOrUpdate(labels map[string]string) error {
 		}.AsData(),
 	}
 
-	err := a.mergeAppUpdates(configMap, labels)
+	if strings.ToLower(os.Getenv("KAPP_EXPERIMENTAL_FQ_CONFIGMAP_NAMES")) == "true" {
```

Comment:
Do we want to have `EXPERIMENTAL` in there? As this might stop a lot of users or make them think twice before they use it. 
(Or is this a way of hinting them to read the docs before they use it.)

Reply:
So I put `EXPERIMENTAL` because I wanted to signal that this feature has consequences that require a manual rollback if they need to downgrade. And also to really examine the docs before they use it.

I do agree though that it will probably stop users from using it as freely as if the word was not part of the flag.

Reply:
It does sound like a good idea to hint them about the docs, I am just thinking that this is something that we would want all the users to use at some point of time, so `EXPERIMENTAL` won't make much sense then (unless we are planning to introduce a new environment variable at a later stage).